synapsis/src/test/java/com/soen/synapsis/integration/ConnectionRepositoryTest.java

```diff
@@ -73,4 +74,20 @@ void itShouldNotFindAcceptedConnectionsByReceiverIDIfUserDoesNotExist() {
         assertTrue(result.isEmpty());
     }
 
+    @Test
+    void itShouldFindPendingConnectionsByReceiverID() {
```

Comment:
should simplify to PendingConnectionsByReceiverID_Success or something simple like this